resources/public/api/conf/5.0/brought_forward_losses_retrieve.yaml

```diff
@@ -1,15 +1,24 @@
 summary: Retrieve a Brought Forward Loss
 description: |
-  This endpoint allows the developer to retrieve an existing brought forward loss amount.
+  
+  This endpoint enables you to retrieve an existing brought forward loss amount. 
   A National Insurance number and Loss ID must be provided.
+
+  <strong>
+  Note: Brought forward losses for the following 2 income sources will not be supported from tax year 2025-26 onwards:
+    <ul style="list-style-type: square;">
```

Comment:
Can you please remove the inline styling as Sion confirmed that it was a mistake in the specs. The bullets style should the the default which is circle. thanks